polaris.shopify.com/content/whats-new/version-11-color.md

```diff
@@ -0,0 +1,80 @@
+title: Version 11 Color
+description: Learn about what changes were introduced to color in Polaris v11.
+imageUrl: /images/updates/new-colors-cover.png
+keywords:
+  - color
+  - tokens
+---
+
+![Preview of new alias tokens](/images/updates/new-colors-cover.png)
+
+## Why are we making changes?
+
+Over the past year we have invested a lot of time and energy into creating a more robust and cohesive token system in Polaris. We have updated typography, spacing, shadows and breakpoints. All of this work has made it easier than ever to build with Polaris and has unlocked new levels of composability within the system itself. We knew we needed to bring this same thinking to our color tokens as well.
+
+These were the main drivers for this update.
+
+1. Create a defined color palette to generate tokens from. This will eliminate color sprawl and ensure a more consistent and cohesive use of color across the admin.
+2. Update key colors to bring more contrast and vibrancy into the admin. Key components like cards often were invisible to merchants because they had so little contrast to the app background itself. This really made it hard for merchants to quickly understand the structure of a given screen.
+3. Create a consistent and thoughtful naming convention that was much simpler and more intuitive for builders to use.
+4. Build an entirely new architecture that allows us to evolve and roll out design changes a lot easier. The previous architecture was rigid and didn’t let us easily cascade changes across the entire system.
+
+## Color palette
+
+We started with the colors themselves. Doing a full audit of the colors used in Polaris. Previously we actually didn’t have a defined color palette to generate tokens from. This led to massive sprawl and inconsistency of hue, saturation, and vibrancy within our colors. We learned that there were actually 22 different gray values in Polaris, some of which were nearly identical to each other, providing no value while introducing added complexity. Every hue had a different range of tints and shades with no consistency or relationship between them.
+
+![image of old color palette](/images/updates/old-palette.png)
+
+We took all these colors and generated a new color palette with 10 shades for each hue. Each shade should be meaningfully different from the ones next to it on the scale. This makes choices purposeful and clear and eliminates the sprawl and muddiness we had before.
+
+These colors power every token provided within the admin. This means that for the first time we can update a single color value within our palette and it will cascade down to every token referencing it, updating the entire admin from a single and simple change.
+
+## Color tokens
+
+There are now two types of color tokens in Polaris, only one of which is actually used by builders.
+
+### Palette tokens
+
+These tokens are not available to use directly, rather, they are where we define the color palette itself. Each color has 10 total shades.
```

Comment:
I am getting stuck on this framing that the color palette is not for use... If it truely wasn't for use it would be completely private and we wouldn't talk about it. The color palette is available to be used and is documented, however there is only one way to use it and it is for creating new alias tokens.